### ISSUE TITLE
Admin-0 world map issue

### ISSUE THREAD

**Issue by levytskyiTaras** [nvkelso/natural-earth-vector#680]
Hello, Is there a way to download the correct .shp and .dbf files for the world map? Because I see just everywhere Crimea belongs to Russia which is not true at all. This Annexation is not recognized by any civilized country in the world.

**Comment by Maxszik** [comment]
Yes you can, it's explained [here](https://www.naturalearthdata.com/about/disputed-boundaries-policy/).
